lib/common.sh

```diff
@@ -294,7 +294,7 @@ determineTool() {
             warn ""
         fi
         if ! <"${DataJSON}" jq  -e '.Go.SupportsModuleExperiment | any(. == "'${ver}'")' &> /dev/null; then
-            err "You are using ${ver}, which does not support the Go modules experiment"
+            err "You are using ${ver}, which does not support the Go modules"
```

Comment:
Not for this PR, but as a follow on one it may be better to list the version of Go that don't support modules. This is similar to how vendoring landed as well. At first the buildpack listed the versions that supported it, then it was switched so that it listed the versions that didn't.

Reply:
Sounds good! I took notes to add this